x-pack/plugins/security_solution/server/lib/detection_engine/routes/utils.test.ts

```diff
@@ -34,17 +33,17 @@ describe.each([
   ['RAC', true],
 ])('utils - %s', (_, isRuleRegistryEnabled) => {
   describe('transformBulkError', () => {
-    test('returns transformed object if it is a boom object', () => {
-      const boom = new Boom.Boom('some boom message', { statusCode: 400 });
-      const transformed = transformBulkError('rule-1', boom);
+    test('returns transformed object if it is a custom error object', () => {
+      const customError = new CustomBadRequestError('some custom error message');
```

Comment:
Same here. I'm thinking maybe you are removing `Boom` from everywhere, correct?

Reply:
Yes, quite right! I assumed we want to remove `Boom` from `security_solution` plugin

Reply:
I initially opened the issue thinking only of our Endpoint related code, but doing the entire plugin is good too. Just make sure you check with the other team to find out if they have Issue to track that activity as well and just close those out when you merge this